Adds C4B-Ansible initial documentation (#853)
chocolatey/docs: input/en-us/c4b-environments/ansible/offline-deployment.md

---
Order: 41
xref: c4b-ansible-offline-deployment
Title: Preparing for an Offline Deployment
Description: How to deploy Chocolatey for Business Ansible Environment without an internet connection
---

## Summary

In some environments, you won't be able to rely on downloadable resources from the internet. For these purposes, you can prepare the Chocolatey for Business Ansible Environment to be deployed without an internet connection.

> ## Requirements

Reviewer: ```suggestion
##  Prerequisites
```

Reviewer: Everywhere else it said this, so being consistent.

> - A Windows machine capable of running Chocolatey
> - An available internet connection
> - PowerShell 5.0+
> - A Chocolatey for Business license or trial license
> - A compatible PFX certificate to use for your deployment

Reviewer: ```suggestion
- A Windows computer capable of running Chocolatey CLI.
- An internet connection.
- PowerShell 5.0+.
- A Chocolatey for Business license or Trial license.
- A compatible PFX certificate to use for your deployment.
```

## Preparing for Offline Deployment

> 1. To begin, download or clone the C4B-Ansible repository to your local machine.

Reviewer: ```suggestion
1. To begin, download or clone the `c4b-ansible` repository to your local machine.
```

> 1. Open a PowerShell terminal and navigate to the repository directory.

Reviewer: ```suggestion
1. Open a PowerShell terminal and navigate to the repository directory you downloaded the files to, above.
```

1. In the PowerShell terminal run the `OfflineInstallPreparation.ps1` script with the following arguments:

>     a. `-LicensePath`: The path to your Chocolatey license file, if it's not installed in the default location
>     a. `-CertificatePath`: The path to your PFX certificate
>     a. `-CertificatePassword`: A `SecureString` of your PFX certificate password.
>         If you do not provide it, you will be prompted to enter it securely.

Reviewer: ```suggestion
    a. `-LicensePath`: The path to your Chocolatey license file, if it's not installed in the default location.
    a. `-CertificatePath`: The path to your PFX certificate.
    a. `-CertificatePassword`: A `SecureString` of your PFX certificate password.
        If you do not provide it, you will be prompted to enter it securely.
```
